src/mptcp.service.in

```diff
@@ -16,4 +16,4 @@ AmbientCapabilities=CAP_NET_ADMIN
 LimitNPROC=1
 
 [Install]
-WantedBy=sockets.target
+WantedBy=multi-user.target
```

Comment:
See the [systemd bootup diagram](https://www.freedesktop.org/software/systemd/man/bootup.html#System%20Manager%20Bootup) for further details.